package.json

```diff
@@ -2,7 +2,7 @@
   "name": "jsonparse",
   "description": "This is a pure-js JSON streaming parser for node.js",
   "tags": ["json", "stream"],
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Since this is a behavior change with new error handling semantics, we need to at lease bump the middle number.